include/circt/Conversion/CombToSMT.h

```diff
@@ -0,0 +1,26 @@
+//===- CombToSMT.h - Comb to SMT dialect conversion ---------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_CONVERSION_COMBTOSMT_H
+#define CIRCT_CONVERSION_COMBTOSMT_H
+
+#include "circt/Support/LLVM.h"
+#include <memory>
```

Comment:
The `Passes.h.inc` include below includes a `unique_ptr`, so `<memory>` has to come from somewhere. Strictly speaking, it still works if we remove this here, but only because `circt/Support/LLVM.h` transitively includes `<memory>` via the casting header included in MLIRs support header. Not sure if we should rely on that.